common/logger/logger_test.go

```diff
@@ -20,6 +20,7 @@ package logger
 import (
 	"fmt"
 	"path/filepath"
+	"runtime"
 	"testing"
 
```

Comment:
pls use this package in another import clause

Reply:
I have added a new import clause.

Reply:
LGTM